frontend/components/App.vue

```diff
@@ -0,0 +1,17 @@
+<template>
+  <div class="h-full flex flex-col">
+    <NavigationBar />
+    <router-view />
+  </div>
+</template>
+
+<script>
```

Comment:
Sometimes is `<script setup>` and sometimes `<script>`. I think that it can be change to `<script setup>`.

Reply:
Almost all `<script>` was changed to `<script setup>`, with one exception MapDisplay.vue which is complicated to change, looks like its need to be complitly rewritten